src/views/InteractiveMap/Escher.vue

```diff
@@ -349,6 +360,37 @@ export default Vue.extend({
       });
       return fluxesFiltered;
     },
+    ecModelFluxes(fluxes) {
+      // Map fluxes from an enzyme-constrained model to the corresponding
+      // escher map, as reaction ids from ecModels have additional prefixes
+      // and/or suffixes.
+      const fluxesMapped = {};
+      Object.keys(fluxes).forEach(rxn => {
+        var newRxn = rxn;
+        var newFlux = fluxes[rxn];
```

Comment:
Prefer to limit scope by using `const` for immutable vars and `let` for mutable ones. (`newFlux` could be immutable if you assign `newFlux * -1` directly on line 389)

Reply:
done in [11f367e](https://github.com/DD-DeCaF/caffeine-vue/pull/171/commits/11f367e0de167fc02d655e776b9dac260cd8b432)